Serialization/Circle.java

```diff
@@ -1,10 +1,10 @@
-public class Circle extends AbstractFigures {
+public class Circle extends AbstractFigure {
 
-    private String Color;
+    private String color;
```

Comment:
You already have color in the abstract class. Add contructor to the AbstractFigure and pass data to AbstractFigure constructor.